scripts/testing/tolerance_test.py

```diff
@@ -37,32 +37,54 @@ def ensure_file(path):
         print("ERROR: Given file is not a file: {0}".format(path))
         sys.exit(1)
 
-
 def parse_args():
     parser = argparse.ArgumentParser(description="Compare two Serac summary files")
+    group = parser.add_mutually_exclusive_group(required=True)
 
     parser.add_argument("--baseline", type=str, required=True,
                         help="Path to baseline summary file")
     parser.add_argument("--test", type=str, required=True,
                         help="Path to test summary file")
-    parser.add_argument("--tolerance", type=float, required=True,
+    group.add_argument("--tolerance", type=str,
                         help="Allowed tolerance amount for individual values")
```

Comment:
Can we add a longer help message that shows possible values here and an example of the JSON file below? If it doesnt look right on the argument help messages(distinct possibility it wont be readable), try adding it to a longer formatted string on the `argparse.ArgumentParser` call (line 41).

Something like:

```
  usage = """
  Compare two Serac summary files
  
  Example usages for --tolerance:
  
  --tolerance=0.001 // used for all field values
  --tolerance=default:0.001::velocity:0.1::displacement:0.234  // defaults to 0.001 with other specific tolerances used
  
  Tolerance JSON file example:
  
  {
 "default": 0.0001,  // used as default
 "velocity": 0.001,   // specific field value tolerance
 "displacement":0.01
}
  """

Reply:
Also need a new section here to explain the usage on this page:

https://serac.readthedocs.io/en/latest/sphinx/dev_guide/testing.html#ats-test-helper-functions